CreeDictionary/CreeDictionary/settings.py

```diff
@@ -63,13 +63,13 @@
     "django_js_reverse",
     # Internal apps
     # TODO: our internal app organization is kind of a mess 🙃
-    "API.apps.APIConfig",
-    "CreeDictionary.apps.CreeDictionaryConfig",
-    "cvd",
-    "search_quality",
-    "phrase_translate",
-    "morphodict.apps.MorphodictConfig",
-    "DatabaseManager",
+    "CreeDictionary.API",
```

Comment:
I love how the comment above was not deleted 💀 💀 💀 

(do not delete that comment; I still agree with it, haha!)